src/docs/markdown/getting-started.md

```diff
@@ -176,6 +186,7 @@ Full range of Caddy functionality | Most common parts of Caddy functionality
 Easy to generate | Easy to craft by hand
 Easily programmable | Difficult to automate
 Extremely expressive | Moderately expressive
+Hard to read | Easy to read 
```

Comment:
So, I disagree on this point. I do think the Caddyfile is easy to read, but I also think the JSON is easy to read. It's definitely more cognitive to _write_ JSON by hand, but not harder to read.

I think we need to keep this table as objective as possible, not subjective, so let's remove this line.